apps/ltn/src/select_boundary.rs

```diff
@@ -151,10 +152,14 @@ impl SelectBoundary {
             }
             Err(err) => {
                 self.last_failed_change = Some((id, self.currently_have_block(app, id)));
-                let label = Text::from(Line(err.to_string()))
-                    .wrap_to_pct(ctx, 15)
-                    .into_widget(ctx);
-                self.left_panel.replace(ctx, "warning", label);
+                // TODO This is some unfortunate complexity spread; we should almost have an
```

Comment:
Leaky abstraction. Making panel visibility happen at the widgetry level could fix this.